bin/pd_ati_monitor.py

```diff
@@ -0,0 +1,61 @@
+import sys
+import subprocess
+import requests
+
+'''
+Usage: python pd_ati_monitor.py usr1@reg_host usr2@portal_host slack_url
+'''
+
+def sendSlack2(text, url):
```

Comment:
`snake_case` is preferred by the python style guide for function and variable names.